cypress/e2e/tldraw/testTldraw.feature

```diff
@@ -0,0 +1,49 @@
+@regression_test
+@stable_test
+Feature: Test tldraw on board
```

Comment:
it should be like `Feature: TLDraw - Test tldraw on board` so in report it will be easily seen